src/Http/Productions/ProductionsWriteController.php

```diff
@@ -56,6 +57,20 @@ public function addEventToProduction(
 
         $this->commandBus->dispatch($command);
 
-        return new Response('', 201);
+        return new Response('', 200);
```

Comment:
Should it actually be `204` (`No content`)?

Reply:
Yes, good point. I'll fix it for all routes in the next PR to avoid unnecessary merges